### DISCUSSION TITLE
Testing compouds

### DISCUSSION THREAD

**Discussion by japserboy** [theOehrly/Fast-F1#317] [General]
HI There,

Happy that everything works so well with testing!

Quick question: what do the compound names refer to in the dataset? During the test day, it looked like there were 7 different compounds avilable to the drivers. Currently, in the data we only see S/M/H/Unknown in the data. How are these being mapped currently? 

Mapping: ['HARD', 'MEDIUM', 'SOFT', 'TEST_UNKNOWN', 'UNKNOWN']
possibilities: C0 C1 C2 C3 C4 C5 and Development

THanks!

**Comment by theOehrly** [comment]
I have absolutely no idea, to be honest. It's the same thing for every testing session that utilizes more tires than normal. The values that FastF1 provides to you are the raw values as received from the API. There are only these five values, and I have no idea how they correspond to the actual compounds.

**Comment by japserboy** [comment]
Okay clear - thanks a lot!

**Comment by MarioFourTwoSix** [comment]
If it's any help, here's the Pirelli infographic https://motorsportmagazine.b-cdn.net/wp-content/uploads/2022/03/Pirelli-2023-F1-testing-tyre-compounds.jpg.webp

**Reply by theOehrly** [reply]
I've seen that before but still it's not easily possible to make sense of the problem. One would need to manually compare, maybe with F1TV, what the compounds are.
